systemd/ocp-custom-domain.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -x
+
+if [ -z $CRC_CLOUD ]; then
+    exit 1
```

Comment:
why `exit 1` may be we should do following to make sure this service is not failed as soon as start for crc side.
```
if [ -z $CRC_CLOUD ]; then
    echo "Not running in crc-cloud mode"
    exit 0
fi
```